fb_hddtemp/README.md

```diff
@@ -0,0 +1,39 @@
+fb_hddtemp Cookbook
+====================
+The `fb_hddtemp` cookbook installs and configures hddtemp, an hard drive
+temperature monitoring utility.
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_hddtemp']['run_daemon']
+* node['fb_hddtemp']['disks']
+* node['fb_hddtemp']['disks_noprobe']
+* node['fb_hddtemp']['interface']
+* node['fb_hddtemp']['port']
+* node['fb_hddtemp']['database']
+* node['fb_hddtemp']['separator']
+* node['fb_hddtemp']['run_syslog']
+* node['fb_hddtemp']['options']
+
+Usage
+-----
+To install hddtemp include `fb_hddtemp`. It will autoprobe the available disks
```

Comment:
Generally we don't list out all the configs if they're documented elsewhere so we're not chasing our tails - this is the whole reason we loop over hashes instead of static definitions.

Since these are all actually SYSCONFIG type stuff (or /etc/default) stuff, I'd move all this to node['fb_hddtemp']['sysconfig'] or similar to match our other cookbooks, and then in attributes do something like:

```
node['fb_hddtemp']['sysconfig'][$CONFIG]
```

And under usage note that we whole-sale put everything int hat hash into /etc/default/hddtemp, "however, we'll summarize current ones here" or some such if you want to include docs.